package.json

```diff
@@ -4,66 +4,66 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^1.3.11",
-    "@date-io/moment": "^2.0.1",
-    "@material-ui/core": "^4.8.3",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "^4.0.0-alpha.42",
+    "@date-io/moment": "^2.10.8",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/pickers": "^3.2.8",
-    "caniuse-lite": "^1.0.30001021",
-    "chart.js": "^2.9.3",
+    "caniuse-lite": "^1.0.30001192",
+    "chart.js": "^2.9.4",
     "classnames": ">=2.2.5",
-    "cropperjs": "^1.5.6",
-    "date-fns": "^2.6.0",
-    "downshift": "^4.0.6",
+    "cropperjs": "^1.5.11",
+    "date-fns": "^2.17.0",
+    "downshift": "^4.1.0",
     "history": "^4.10.1",
     "http-proxy-middleware": "^0.20.0",
-    "interweave": "^12.1.1",
-    "jwt-decode": ">=2.2.0",
-    "lodash": ">=4.17.11",
-    "luxon": "^1.21.3",
-    "moment": ">=2.24.0",
+    "interweave": "^12.7.0",
+    "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
+    "luxon": "^1.26.0",
+    "moment": "^2.29.1",
     "prop-types": ">=15.6.0",
-    "react": "^16.12.0",
-    "react-big-calendar": ">=0.21.0",
-    "react-chartjs-2": ">=2.7.4",
-    "react-cropper": ">=1.0.1",
+    "react": "^16.14.0",
+    "react-big-calendar": "^0.32.0",
+    "react-chartjs-2": "^2.11.1",
+    "react-cropper": "^2.1.7",
```

Comment:
Also a change in major version here. We might want to test that ID uploader still works fine.